src/components/map/Cluster.js

```diff
@@ -14,18 +14,34 @@ const MIN_CLUSTER_DIAMETER = 30
  */
 const MAX_CLUSTER_DIAMETER = 100
 
-// TODO: Cluster styling/icon that Siraj wants
 const ClusterContainer = styled.button`
   width: ${(props) => props.diameter}px;
   height: ${(props) => props.diameter}px;
   display: flex;
   justify-content: center;
   align-items: center;
+  border: none;
   border-radius: 100%;
-  background: rgba(35, 74, 201, 0.214);
+  color: white;
+  background: ${({ theme }) => theme.blue};
+  box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.12);
   transform: translate(-50%, -50%);
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+  }
 `
 
+/**
+ * Helper function to round and format cluster labels
+ * @function formatClusterLabel
+ * @param {number} count - The total number of locations within this cluster
+ * @return {string} - The rounded number label
+ */
+const formatClusterLabel = (count) =>
+  count < 1000 ? count : `${Math.ceil(count / 1000).toString()}K`
```

Comment:
You can actually omit `.toString()` here, since it's automatically called during template string formatting. That's why using it is so convenient!